tools/clang/unittests/HLSL/FileCheckerTest.cpp

```diff
@@ -66,417 +66,433 @@ static string trim(string value) {
   return value;
 }
 
-    FileRunCommandPart::FileRunCommandPart(const std::string &command, const std::string &arguments, LPCWSTR commandFileName) :
-      Command(command), Arguments(arguments), CommandFileName(commandFileName) { }
-    FileRunCommandPart::FileRunCommandPart(FileRunCommandPart && other) :
-      Command(std::move(other.Command)),
-      Arguments(std::move(other.Arguments)),
-      CommandFileName(other.CommandFileName),
-      RunResult(other.RunResult),
-      StdOut(std::move(other.StdOut)),
-      StdErr(std::move(other.StdErr)) { }
-
-    void FileRunCommandPart::Run(const FileRunCommandPart *Prior) {
-      bool isFileCheck =
-        0 == _stricmp(Command.c_str(), "FileCheck") ||
-        0 == _stricmp(Command.c_str(), "%FileCheck");
-      // For now, propagate errors.
-      if (Prior && Prior->RunResult) {
-        if (isFileCheck) {
-          RunFileChecker(Prior);
-        } else {
-          StdErr = Prior->StdErr;
-          RunResult = Prior->RunResult;
-        }
-        return;
-      }
-
-      // We would add support for 'not' and 'llc' here.
-      if (isFileCheck) {
-        RunFileChecker(Prior);
-      }
-      else if (0 == _stricmp(Command.c_str(), "StdErrCheck")) {
-        RunStdErrChecker(Prior);
-      }
-      else if (0 == _stricmp(Command.c_str(), "tee")) {
-        RunTee(Prior);
-      }
-      else if (0 == _stricmp(Command.c_str(), "%dxc")) {
-        RunDxc(Prior);
-      }
-      else if (0 == _stricmp(Command.c_str(), "%dxv")) {
-        RunDxv(Prior);
-      }
-      else if (0 == _stricmp(Command.c_str(), "%opt")) {
-        RunOpt(Prior);
-      }
-      else if (0 == _stricmp(Command.c_str(), "%D3DReflect")) {
-        RunD3DReflect(Prior);
-      }
-      else {
-        RunResult = 1;
-        StdErr = "Unrecognized command ";
-        StdErr += Command;
-      }
-    }
+FileRunCommandPart::FileRunCommandPart(const std::string &command, const std::string &arguments, LPCWSTR commandFileName) :
```

Comment:
Sorry for the noise, I just fixed the indentation of all of this code. The only changes are to `FileRunCommandPart::Run` and `FileRunCommandPart::RunXFail`.